enacts/calc.py

```diff
@@ -121,6 +126,126 @@ def synthesize_enacts(variable, time_res):
     var_name = variable
     return xrds[var_name]
 
+
+def sql2GeoJSON(shapes_sql, db_sql):
+    """ Form a GeoJSON dict from sql request to a database
+
+    Parameters
+    ----------
+    shapes_sql: str
+        sql request
+    db_sql: dict
+        dictionary with host, port, user and dbname information
```

Comment:
This is not SQL, so I don't understand why the variable name is `db_sql`. `db_config`?